hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroWriteSupport.java

```diff
@@ -35,17 +36,19 @@
 /**
  * Wrap AvroWriterSupport for plugging in the bloom filter.
  */
-public class HoodieAvroWriteSupport extends AvroWriteSupport {
+public class HoodieAvroWriteSupport<T> extends AvroWriteSupport<T> {
```

Comment:
Seems like this type parameter `<T>` is not needed?

Reply:
I think, if users are over-ridding write() method from the AvroWriteSupport class, they require the generics to be passed.

Reply:
Got it.